docs/design/UseOfOptional.md

```diff
@@ -2,19 +2,15 @@
 
 ## Use of Optional
 
-This page describes general guidelines of how we use
-`java.util.Optional`.
+This page describes a general guideline for the use of
+`java.util.Optional` in the AWS SDK for Java 2.x.
 
-- `Optional` should be used when it isn't obvious to a caller whether a
-  result will be null.
-- `Optional` must not be used when the result will never be
-  `Optional.empty()`.
-- `Optional` must not be used for instance (in `@SdkPublicApi`s).
-- `Optional` must not be used for parameters (in `@SdkPublicApi`s).
-- `Optional` must not be used when the caller may know whether a value
-  is present. As a consequence:
-  - `Optional` must not be used in classes with "setters" and "getters",
-    like Builders or POJOs.
+- `Optional` SHOULD be used when it isn't obvious to a caller whether a
+  result will be null, e.g., 
+- `Optional` MUST not be used when the result will never be null.
```

Comment:
When the statement contains `not`, I think it should also be caps, so `MUST not` => `MUST NOT`